src/homework19/Main.java

```
@@ -0,0 +1,91 @@
+package homework19;
+
+import java.awt.*;
+import java.util.*;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
```

in main method we need to catch all Exceptions
its the last place where you can catch them, is Exception appear here, program will stop what is not preferable